config/charts/templates/kube-prometheus-stack.tpl

```diff
@@ -60,9 +66,18 @@ alertmanager:
       webhook_configs:
       - url:  "http://matrixbot:8080/skill/alertmanager/webhook"
     {{#if opsgenieEnabled}}
+    - name: heartbeats
+      webhook_configs:
+      - http_config:
+          basic_auth:
+            password: {{ env "OPSGENIE_TOKEN" | default "opsgenie_token" }}
+        url: https://api.eu.opsgenie.com/v2/heartbeats/{{ deploymentName }}/ping
```

Comment:
@pampatzoglou has this endpoint to be configured on opsgenie ?

Reply:
no we need to manually goto the webUI of opsgenie and set them up https://web3foundation.app.eu.opsgenie.com/settings/heartbeat